xain_fl/coordinator/legacy_participant.py

```diff
@@ -1,8 +1,9 @@
 """Module implementing the networked Participant using gRPC.
 """
 
-# TODO: Currently, the participant is completely ignored in the mypy.ini, since the participant
-# will be removed anyways in XP-208.
+# TODO: this is an outdated version of the Participant state machine.
```

Comment:
Do we need to keep legacy code?

Reply:
This: https://github.com/xainag/xain-sdk/pull/13 contains the most current code changes
This: https://github.com/xainag/xain-sdk/pull/21 will delete it from xain-sdk

So it need to be moved here. So this file or another needs to contain the most current version. Ergo it was outdated but is now soon to be replaced

Reply:
i'll solve it as part of https://xainag.atlassian.net/browse/XP-373 let's merge this PR first